### PR TITLE
remove deprecations mssql

### PR DESCRIPTION
**Pull request by Bowrna**
Related: #39485
Removing deprecated MsSqlOperator from docs and tests and replacing with SQLExecuteQueryOperator

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Review by dirrao — APPROVED** [review]
LGTM

File: docs/apache-airflow-providers-microsoft-mssql/operators.rst [hunk 1]
```diff
@@ -17,26 +17,29 @@
...
-Common Database Operations with MsSqlOperator
-------------------------------------------------
+.. warning::
+    Previously, MsSqlOperator was used to perform this kind of operation. But at the moment MsSqlOperator is deprecated and will be removed in future versions of the provider. Please consider to switch to SQLExecuteQueryOperator as soon as possible.
 
-To use the mssql operator to carry out SQL request, two parameters are required: ``sql`` and ``mssql_conn_id``.
+Common Database Operations with SQLExecuteQueryOperator
+-------------------------------------------------------
+
+To use the mssql operator to carry out SQL request, two parameters are required: ``sql`` and ``conn_id``.
```
Comment (RNHTTR) [inline docs/apache-airflow-providers-microsoft-mssql/operators.rst]:
```suggestion
To use the SQLExecuteQueryOperator to execute SQL queries against an MSSQL database, two parameters are required: ``sql`` and ``conn_id``.
```

**Review by RNHTTR — CHANGES_REQUESTED** [review]
I think this should refer to the SQLExecuteQueryOperator?

File: docs/apache-airflow-providers-microsoft-mssql/operators.rst (hunk 1 above)
Comment (Bowrna) [reply docs/apache-airflow-providers-microsoft-mssql/operators.rst]: yes @RNHTTR

**Review by Taragolis — APPROVED** [review]

**Review by RNHTTR — APPROVED** [review]

**Review by vincbeck — APPROVED** [review]

**Comment by vincbeck** [comment]
Please resolve conflict

**Comment by Bowrna** [comment]
I have resolved the conflicts in this PR

**Review by potiuk — APPROVED** [review]

**Pull request merged**